app/models/talentlms/facade.rb

```diff
@@ -10,42 +10,111 @@ def initialize
       @api = Config.first
     end
 
+    def talentlms_user_record_by_id(id)
+      result = @api.post('users', { id: id })
+      result
+    rescue RuntimeError => e
+      raise e unless e.message.include?('The requested user does not exist')
+    end
+
+    def talentlms_user_record_by_email(email)
+      result = @api.post('users', { email: email })
+      result
+    rescue RuntimeError => e
+      raise e unless e.message.include?('The requested user does not exist')
+    end
+
+    def talentlms_user_record_by_username(username)
+      result = @api.post('users', { username: username })
+      result
+    rescue RuntimeError => e
+      raise e unless e.message.include?('The requested user does not exist')
+    end
+
+    def lms_username(user)
```

Comment:
Can you add a brief comment describing what this is doing and specifically what the dev_offset is for?

Reply:
added in https://github.com/greenriver/hmis-warehouse/pull/4830/commits/8325cf3128615c0819c2a15ea6edafd7b64bc8ed